tests/test_selectors.py

```diff
@@ -25,3 +25,18 @@ def test_selectors(selector, expected_value):
     selected_value = selector.select(values, smaller_is_better=False)
 
     assert selected_value == expected_value
+
+
+def test_random_selector():
+    """
+    Check that the :class:`RandomSelector`, when ran multiple times, selects approximately the expected value for known
+    examples.
+    """
+    selector = peppr.RandomSelector(k=5)
```

Comment:
The random seed could be used in this test to make it deterministic

Reply:
Addressed in af4a29a889e3a24b9d2df186bfdd95834d0a9e5d.